internal/config/config.go

```diff
@@ -17,27 +17,35 @@ import (
 )
 
 const (
-	filesystemBackend = "filesystem"
-	memoryBackend     = "memory"
+	filesystemBackend   = "filesystem"
+	memoryBackend       = "memory"
+	eventFinalize       = "finalize"
+	eventDelete         = "delete"
+	eventMetadataUpdate = "metadataUpdate"
 )
 
 type Config struct {
-	Seed               string
-	publicHost         string
-	externalURL        string
-	allowedCORSHeaders []string
-	scheme             string
-	host               string
-	port               uint
-	backend            string
-	fsRoot             string
+	Seed                string
+	publicHost          string
+	externalURL         string
+	allowedCORSHeaders  []string
+	scheme              string
+	host                string
+	port                uint
+	backend             string
+	fsRoot              string
+	eventPubsubProjecID string
+	eventPubsubTopic    string
+	eventPrefix         string
+	eventList           []string
```

Comment:
Can you move these fields into a dedicated struct for events?

Reply:
(also, with the growing number of flags we have, we'd should probably consider adding some form of config file, not a blocker for this PR though, definitely something I'll have to figure out later)